src/test/java/io/reactivex/internal/operators/completable/CompletableAndThenTest.java

```diff
@@ -69,39 +63,4 @@ public void andThenMaybeError() {
             .assertError(RuntimeException.class)
             .assertErrorMessage("bla");
     }
-
-    @Test
-    public void andThenNoInterrupt() throws InterruptedException {
```

Comment:
Why was this test removed without explanation?

Reply:
I moved it to CompletableAndThenCompletableTest since it specifically tested `.andThen(Completable)`.